jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/MainFragment.java

```diff
@@ -13,8 +12,7 @@
 
 public class MainFragment extends Fragment {
 
-    // The main activity.
-    MainActivity main_activity;
+    MainActivity mainActivity;
```

Comment:
Thought I saw you asking a question about the naming here somewhere. The ~convention for camelCase vs underscore_case in Android (in our team) is that ids and names constructed for us on the fly are underscore_case, otherwise camelCase